### PR TITLE
prepare for geo-0.24.0 release

### PR DESCRIPTION
**Pull request by michaelkirk**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/main/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

~~Note: this depends on #989 being released first~~ Update: released!

Anything else we should be waiting on before a geo-0.24.0 release?

### CODE REVIEW

File: geo-types/CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changes
 
-## unreleased
```
Comment (frewsxcv) [inline geo-types/CHANGES.md:3]: this is the wrong changelog change

**Review by frewsxcv — APPROVED** [review]

File: geo-types/CHANGES.md (hunk 1 above)
Comment (michaelkirk) [reply geo-types/CHANGES.md:3]: Good catch! I've updated the geo CHANGES file.

(note that this PR still contains updates to the geo-types CHANGES file by virtue of it being based on #989)

**Comment by michaelkirk** [comment]
I'll plan to wait on #991  (which seems straight forward) before releasing geo 0.24.0

**Comment by michaelkirk** [comment]
published: https://crates.io/crates/geo/0.24.0
tagged: https://github.com/georust/geo/releases/tag/geo-0.24.0

**Pull request merged**